projects/cms-monit-micro-services.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: To develop microservice architecture for CMS HTCondor Job Monitoring
+
+postdate: 29-03-2023
+categories:
+  - Any   Analytics and Open science 
```

Comment:
Hi @brij01 - sorry I missed this pr when made.

Could you turn this into valid yaml - to first order that means removing the "any" and then making the other keywords into a list.

Reply:
Done

Reply:
@davidlange6  Kindly check now.